docs/source/model_design/risk_factors.rst

```diff
@@ -36,6 +36,18 @@ What is a risk factor?
 What is a causal relationship?
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
+Counfounding
```

Comment:
I think this should just be 'Confounding', other than that this is great!

Reply:
oops :). I will fix. Thanks!